packages/assets-controllers/src/MultichainTokensRatesController/constant.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  BtcAccountType,
+  SolAccountType,
+  EthAccountType,
+  type KeyringAccountType,
+} from '@metamask/keyring-api';
+
+export const MAP_SWIFT_ISO4217: Record<KeyringAccountType, string> = {
+  [SolAccountType.DataAccount]: 'swift:0/iso4217:SOL',
+  [BtcAccountType.P2wpkh]: 'swift:0/iso4217:BTC',
+  [EthAccountType.Eoa]: 'swift:0/iso4217:ETH',
+  [EthAccountType.Erc4337]: 'swift:0/iso4217:ETH',
+};
```

Comment:
This mapping doesn't mean anything. the iso-4217 doesn't reference any cryptocurrencies ([ref](https://github.com/ChainAgnostic/namespaces/pull/132/files)).